### ISSUE TITLE
Reduce the width of the Navigation Panel

### ISSUE THREAD

**Issue by blackrosedragon2** [Adversarial-Deep-Learning/adversarial-deep-learning.github.io#11]

**Comment by arihantbansal** [comment]
I will take this up.

**Comment by arihantbansal** [comment]
![Screenshot 2021-10-09 121032](https://user-images.githubusercontent.com/17180950/136647231-7732be78-da29-45bf-9ab7-c8c6e30df2ee.png)
Does this look good?

**Comment by harsh4799** [comment]
Yup looks good, check the ipynb file on ch5 as well and then create a PR

**Comment by arihantbansal** [comment]
> Yup looks good, check the ipynb file on ch5 as well and then create a PR

The ipynb is messing up the styles for the rest of the page. I'm working on a fix for it.
